docs/usage.rst

```diff
@@ -0,0 +1,256 @@
+        .add_univariate_analysis(use_columns=["Name", "Sex", "Age"])
+    )
+
+
+Subsections
+***********
+
+Some sections are made of subsections. For those, you can can configure which subsections are be included.
+
+.. code-block:: python
+
+    import edvart
+    from edvart.report_sections.dataset_overview import Overview
+
+
+    df = edvart.example_datasets.dataset_titanic()
+    report = edvart.Report(df)
+
+    report.add_overview(
+        subsections=[
+            Overview.OverviewSubsection.QuickInfo,
+            Overview.OverviewSubsection.DataPreview,
+        ]
+    )
+
+
+.. _verbosity:
+
+Verbosity
+~~~~~~~~~
+
+A :py:class:`~edvart.report.Report` can be exported to a Jupyter notebook containing
+the code which generates the report. The code can be exported with different levels of detail,
+referred to as *verbosity*.
+
+It can be set on the level of the whole report or on the level of each
+section or subsection separately (see :ref:`sections-config`).
+
+Specific verbosity overrides general verbosity, i.e. the verbosity set on a
+subsection overrides the verbosity set on a section, which overrides
+the verbosity set on the report.
+
+EDVART supports three levels of verbosity:
+
+LOW
+   High level functions for whole sections are exported, i.e. the output
+   of each section is generated by a single function call.
+   Suitable for small modifications such as changing parameters of the functions,
+   adding commentary to the report, adding visualizations which are not in EDVART, etc.
+
+MEDIUM
+   For report sections which consist of subsections, each subsection is
+   exported to a separate function call.
+   Same as LOW for report sections which do not consist of subsections.
+
+HIGH
+   The definitions of (almost) all functions are exported.
+   The functions can be modified or used as a starting point for custom analysis.
+
+
+Examples
+********
+
+.. code-block:: python
+
+    # Set default verbosity for all sections to Verbosity.MEDIUM
+    import edvart
+    from edvart import Verbosity
+
+
+    df = edvart.example_datasets.dataset_titanic()
+    edvart.DefaultReport(df, verbosity=Verbosity.MEDIUM).export_notebook("test-export.ipynb")
+
+
+.. code-block:: python
+
+    import edvart
+    from edvart import Verbosity
+
+
+    # Set report verbosity to Verbosity.MEDIUM but use verbosity Verbosity.HIGH for univariate analysis
+    df = edvart.example_datasets.dataset_titanic()
+    edvart.DefaultReport(
+        df,
+        verbosity=Verbosity.MEDIUM,
+        verbosity_univariate_analysis=Verbosity.HIGH,
+    ).export_notebook("exported-report.ipynb")
+
+
+Exporting Notebooks to HTML
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A :py:class:`~edvart.report.Report` can be directly exported
+to HTML via the :py:meth:`~edvart.report.ReportBase.export_html` method.
+
+Jupyter notebooks can be exported to other formats including HTML, using a tool
+called `jupyter nbconvert` (https://nbconvert.readthedocs.io/en/latest/).
```

Comment:
Couldn't this be merged with section `Export a Report to HTML`? Not sure if it's necessary to have this section even under `Customizing the Report`.

Reply:
Good idea, it fits there well. Moved under the example, and added a note when exporting using `jupyter nbconvert` can be useful (to create HTML from exported notebook).